src/Stott.Optimizely.RobotsHandler/Configuration/RobotsServiceExtensions.cs

```diff
@@ -42,4 +43,16 @@ public static class ServiceExtensions
 
         return serviceCollection;
     }
+
+    public static IApplicationBuilder AddRobotsHandler(this IApplicationBuilder builder)
```

Comment:
Conventions would have this method be called `UseRobotsHandler`.  Typically methods called during the ConfigureServices stage of the app start with `Add` while those in the Configure stage use the prefix `Use`